Blazor RC2 Updates 8.0 (#30576)
dotnet/AspNetCore.Docs: aspnetcore/blazor/tooling.md

* Install the latest version of [Visual Studio Preview](https://visualstudio.microsoft.com/vs/preview/) with the **ASP.NET and web development** workload.

> * Create a new project:
>   * For a Blazor Web App experience (*recommended*), choose the **Blazor Web App** template. Select **Next**.
>   * For a Blazor WebAssembly experience, choose the **Blazor WebAssembly Standalone App** template. Select **Next**.

Reviewer: The main point of these bullets seems to be to indicate the available Blazor templates. How about we just list the templates by name here and have a sentence about what each template is for?

```suggestion
* Create a new project using one of the available Blazor templates:
  * **Blazor Web App** (*recommended*): Creates a Blazor web app that supports both server and client-side rendering with rich interactivity. 
  * **Blazor WebAssembly Standalone App**: Creates a standalone client web app that can be deployed as a static site.
```

:::moniker range=">= aspnetcore-8.0"

* For a Blazor Web App in the **Additional information** dialog:

<!-- UPDATE 8.0 For RTM, the dropdown list label changes to 
                **Interactive render mode** -->

>   * **Interactivity type** dropdown list:

Reviewer: Just a heads up that the label for this dropdown will change to "Interactive render mode" for .NET 8 GA.

>     * Interactivity with server rendering is enabled by default with the **Server** option.

Reviewer: ```suggestion
    * Interactive server rendering is enabled by default with the **Server** option.
```

    * To only enable interactivity with client rendering, select the **WebAssembly** option.

>     * To enable interactive server rendering followed by interactive WebAssembly rendering, select the **Auto (Server and WebAssembly)** (automatic) render mode option.

Reviewer: ```suggestion
    * To enable both interactive server and client rendering and the ability to automatically switch between them at runtime, select the **Auto (Server and WebAssembly)** (automatic) render mode option.
```

    * If interactivity is set to `None`, the generated app has no interactivity (static server rendering only).

    The automatic (Auto) render mode initially uses interactive server rendering while the .NET app bundle and runtime are download to the browser. After the .NET WebAssembly runtime is activated, the render mode switches to interactive WebAssembly rendering.

    By default, the Blazor Web App template enables both static and interactive server rendering using a single project. If you also enable interactive WebAssembly rendering, the project includes an additional client project (`.Client`) for your WebAssembly-based components. The built output from the client project is downloaded to the browser and executed on the client. Any components using the WebAssembly or automatic render modes must be built from the client project.

    For more information, see <xref:blazor/components/render-modes>.

  * **Interactivity location** dropdown list:

>     * **Per page/component**: The default sets up the app for interactivity set per page or per component.

Reviewer: ```suggestion
    * **Per page/component**: The default sets up the app for interactivity per page or per component.
```

>     * **Global**: Selecting this option sets interactivity for entire app in the `App` component by specifying a render mode for the top-level `HeadOutlet` and `Routes` components. Setting the interactivity on these components propagates the interactivity to all of the child components in the app.

Reviewer: ```suggestion
    * **Global**: Selecting this option sets up interactivity globally for entire app.
```

</PropertyGroup>
```

<!-- UPDATE 9.0 Remove note when https://github.com/dotnet/runtime/issues/89302 is fixed -->

> [!IMPORTANT]

> > Disabling SIMD isn't supported. For more information, see [SIMD cannot be disabled (dotnet/runtime #89302)](https://github.com/dotnet/runtime/issues/89302).

Reviewer: If disabling SIMD currently isn't supported should we remove this section? Why tell the user how to disable it and then immediate say it can't be disabled?
